@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1627,17 +1638,19 @@ export class TextElementsRenderer {
                 this.m_screenCollisions
             );
             if (result === PlacementResult.Invisible) {
+                // Reset the icon and keep it from popping up before fading in.
+                iconInvisible = true;
                 iconRenderState.reset();
-
-                if (placementStats) {
+                iconRenderState.opacity = 0;
```

Comment:
this line is only reached if placementStats is defined, i guess that was unintended.
Anyway, iconRenderState.reset() should take care of setting the opacity to 0. So far it sets it to 1, i don't understand why.